packages/loopring_v3/contracts/impl/libtransactions/NewAccountTransaction.sol

```diff
@@ -16,45 +14,46 @@ import "../../lib/SignatureUtil.sol";
 library NewAccountTransaction
 {
     using BytesUtil            for bytes;
-    using FloatUtil            for uint;
-    using MathUint             for uint;
     using SignatureUtil        for bytes32;
 
     bytes32 constant public NEWACCOUNT_TYPEHASH = keccak256(
         "NewAccount(uint24 accountID,address owner,uint256 publicKey,uint256 walletHash)"
     );
 
-    /*event NewAccountCreated(
+    /*event AccountCreated(
         address  indexed owner,
         uint             publicKey,
         uint             walletHash
     );*/
 
     function process(
-        ExchangeData.State storage S,
-        ExchangeData.BlockContext memory ctx,
-        bytes memory data,
-        bytes memory auxiliaryData
+        ExchangeData.State        storage S,
+        ExchangeData.BlockContext memory  ctx,
+        bytes                     memory  data,
+        bytes                     memory  auxiliaryData
         )
         internal
         returns (uint /*feeETH*/)
     {
-        uint offset = 1;
+        // uint offset = 1;
 
         // Extract the data from the tx data
         //uint24 payerAccountID = data.toUint24(offset);
-        offset += 3;
+        // offset += 3;
         //uint16 feeTokenID = data.toUint16(offset);
-        offset += 2;
+        // offset += 2;
         //uint fee = uint(data.toUint16(offset)).decodeFloat(16);
-        offset += 2;
-        uint24 newAccountID = data.toUint24(offset);
+        // offset += 2;
+
+        uint offset = 8;
```

Comment:
I prefer not to do it like this because the `8` is now a special value that comes from nowhere.